ext/jetbrick/src/main/java/org/mvcspec/ozark/ext/jetbrick/JetbrickViewEngine.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2017 Ivar Grimstad (ivar.grimstad@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mvcspec.ozark.ext.jetbrick;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.mvc.engine.ViewEngineContext;
+import javax.mvc.engine.ViewEngineException;
+import javax.servlet.ServletContext;
+
+import org.mvcspec.ozark.engine.ViewEngineBase;
+
+import jetbrick.template.JetEngine;
+import jetbrick.template.JetTemplate;
+import jetbrick.template.web.JetWebEngine;
+
+/**
+ * @author Daniel Dias
+ */
+@ApplicationScoped
+public class JetbrickViewEngine  extends ViewEngineBase {
+
+	private  JetEngine jetEngine = JetWebEngine.getEngine();
+
+	@Inject
+	private ServletContext servletContext;
+
+	@Override
+	public boolean supports(String view) {
+		return view.endsWith(".jetx");
+	}
+
+	@Override
+	public void processView(ViewEngineContext context) throws ViewEngineException {
+		jetEngine = JetWebEngine.create(servletContext);
```

Comment:
What do you think about creating the engine once. Looks like this **could** be an expensive operation and therefore creating it once could make sense. We could simply use a `@PostConstruct` method for this and convert the engine into a field.

Reply:
Hi @chkal ,

in case it should look like this:

``` java
private JetEngine jetEngine;

@PostConstruct
public void setup() {
     jetEngine = JetWebEngine.getEngine();
     jetEngine = JetWebEngine.create(servletContext);
    }
```

Reply:
Hmmm. What's the difference between `JetWebEngine.getEngine()` and `JetWebEngine.create(servletContext)`? Looks redundant to me.

Reply:
I removed this ```JetWebEngine.getEngine ()``` and functioned normally. You will not need this instruction.

from the documentation I saw this section:

>In Web integration mode, JetEngine is created by JetWebEngine.create (...) (singleton mode), so that we can get JetEngine

had also seen as an example in an extension to Struts 2 that uses it in line 43: 

https://github.com/subchen/jetbrick-template-2x/blob/master/jetbrick-template-struts/src/main/java/jetbrick/template/web/struts/JetTemplateResult.java

Reply:
Ok, great! So just push the changes and we can merge the PR. Thanks.

Reply:
done. thank you.